litellm/integrations/opik/utils.py

```diff
@@ -0,0 +1,97 @@
+import os
+import time
+from typing import Optional, Final, Dict
+import configparser
+from litellm.types.utils import ModelResponse
+
+CONFIG_FILE_PATH_DEFAULT: Final[str] = "~/.opik.config"
+
+def create_uuid7():
```

Comment:
You probably copied this from somewhere, if yes, would be good to add a link to it and add a comment about its license

Reply:
How did you know :)